webdriver/tests/element_click/link.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+
+from tests.support.asserts import assert_error, assert_success
+from tests.support.inline import inline
+
+def click(session, element):
+    return session.transport.send("POST", "session/{session_id}/element/{element_id}/click"
+                                  .format(session_id=session.session_id,
+                                          element_id=element.id))
+
+
+# 14.1 Element Click Link Element Tests Step 7
+
+def test_click_numbers_link(session):
```

Comment:
s/_click//g in all the test titles, since this is implied by the package name.

Reply:
fixed